src/commands/index.ts

```diff
@@ -43,6 +44,7 @@ export const commands = {
     "modules:disable": ModulesDisable,
     "service-logs": ServiceLogs,
     "troubleshoot:analyse": TroubleshootAnalysis,
+    "troubleshoot:proxy-status": TroubleshootProxyStatus,
```

Comment:
I think the idea of Troubleshooting is that each item is an implementation of `AnalysisTask` much in the same as: https://github.com/companieshouse/chs-dev/blob/main/src/run/troubleshoot/analysis/ServicesInLiveUpdateConfiguredCorrectlyAnalysis.ts and then run as a pack. Something to be considered is whether the `analysis` command takes flag(s) to identify the task(s) to run - but I'd propose doing this after a few more tasks added